### PR TITLE
corrected typo and added parameter for cluster (#1)

### PR DESCRIPTION
**Pull request by Zacmatix**
## Description

I corrected a typo and added a parameter to the task run
## Motivation and Context

It both solves a bug ( the typo was critical) and allows for more configuration. It was blocking for us at some point to not be able to run tasks on clusters other than the default one.
## Have you tested this? If so, how?

I ran my code (not sharable) and it worked just fine.

### CODE REVIEW

**Comment by Zacmatix** [comment]
@jfeala

File: tox.ini [hunk 1]
```diff
@@ -12,6 +12,7 @@ deps=
   nose<2.0
   unittest2<2.0
   boto<3.0
```
Comment (Tarrasch) [inline tox.ini]: I know nothing about boto. But is it possible to remove `boto` dependency now that you introduce `boto3`?

**Comment by wblancqu** [comment]
We are facing the same issue, any progress on merging this PR?

Kr, Wouter

**Comment by vanhove** [comment]
This PR #2045 fixes the same issue.

**Pull request closed without merging**